common/lib/api-client/models.js

```diff
@@ -117,6 +117,18 @@ module.exports = {
       collectionPath: 'reference/suppliers',
     },
   },
+  prison_transfer_reason: {
+    attributes: {
+      id: '',
+      type: '',
```

Comment:
As we discussed we shouldn't include `type` as an attribute as its reserved in JSON:API. Same goes for `id`. They'll get put onto the base object by the [devour client library](https://github.com/twg/devour).